src/services/project.service.ts

```diff
@@ -42,11 +42,38 @@ export const createProject = (
     });
 };
 
-export const getMentors = () => {
-  return axios.get(URLMentors).then((res) => {
-    return res;
-  });
-};
+  export const getMentors = () => {
+    return axios.get(URLMentors)
+      .then((res) => {
+        return res;
+      })
+  };
+
+
+  export const createTeamProject = (
+    name: string,
+    mentorId: string,
+    teamId: string,
+    language: string[],
+    content: string,
+    description: string,
+  ) => {
+    return axios
+      .post(URL, { 
+        name,
+        mentorId,
+        teamId,
+        language,
+        content,
+        description
+      })
```

Comment:
tak się zastanawiam, da się zamiast przekazywać każdą wartość po kolei to przekazywać zmienną np. payload która przechowuje obiekt z tymi danymi
```js
interface IPayload =  name: string,
    mentorId: string,
    teamId: string,
    language: string[],
    content: string,
    description: string,
}
 export const createTeamProject = (  payload: IPayload  ) => {
    return axios
      .post(URL, { ...data })
```

Reply:
Próbowałam to wcześniej zmienić, ale krzyczał na mnie plik AddNewTeamProject.tsx, dlatego na razie zostawiłam tak :)